File: Server/Entities/Section/Section.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace OpenWiki.Server.Entities {
    public class Section {
        [Key]
        public long ID { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
    }
}

File: ClientApp/src/app/models/Section.ts
export class Section {
    id: number;
    title: string;
    content: string;
  
    constructor(object: any) {
      if(object.id != null){
        this.id = object.id;
      }
      this.title = object?.title || '';
      this.content = object?.content || '';
    }
  }
  